### PR TITLE
chore(website): Add tailscale partner blog

### PR DESCRIPTION
**Pull request by yevgenypats**
This is ready for review but I'll wait before merging until Tailscale will post on their website (I'll need to share this PR for their review as well)

### CODE REVIEW

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+```
+kind: source
+# Common source-plugin configuration
+spec:
+  name: tailscale
+  path: cloudquery/tailscale
+  version: "VERSION_SOURCE_TAILSCALE"
+  tables: ["tailscale_devices"]
+  destinations: ["postgresql"]
+  # Stripe specific configuration
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:45]:
```suggestion
  # Tailscale specific configuration
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  spec:
+    client_id: "YOUR_CLIENT_ID"
+    client_secret: "${TAILSCALE_CLIENT_SECRET}"
+    tailnet: "cloudquery.io"
+```
+
+## Example Queries
+
+Let's look at a few useful queries we can do just with the tailscale tables.
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx]:
```suggestion
Let's look at a few queries we can run on the Tailscale tables.
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+title: Announcing the CloudQuery Tailscale Source Plugin
+tag: announcement
+date: 2023/02/07
+description: >-
+  This tutorial will show you the basic of ingesting your Tailscale configuration to any database/datalake with CloudQuery
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:6]:
```suggestion
  This tutorial will show you the basic of ingesting your Tailscale configuration to any database/data lake with CloudQuery
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Introduction
+
+Tailscale source plugin for CloudQuery is now officially maintained by Tailscale - Big thanks for the Tailscale team!.
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:16]:
```suggestion
The Tailscale source plugin for CloudQuery is now officially maintained by Tailscale—big thanks to the Tailscale team!
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    tailnet: "cloudquery.io"
+```
+
+## Example Queries
+
+Let's look at a few useful queries we can do just with the tailscale tables.
+
+### Unseen devices
+
+Find all devices that weren't seen more than 30 days:
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:58]:
```suggestion
Find all devices that weren't seen for more than 30 days:
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+```sql
+select name, id, last_seen from tailscale_devices where last_seen < NOW() - INTERVAL '30 DAY';
+```
+
+```
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:64]:
```suggestion
```text
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+ - [`tailscale_dns_searchpaths`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_dns_searchpaths.md) table
+ - [`tailscale_keys`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_keys.md) table
+
+Let's look at a few useful queries
+
+## Configuration
+
+First let's quickly look at the source configuration so the plugin can authenticate with the new OAuth Client
+
+```
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:36]:
```suggestion
```yaml copy
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+## Example Queries
+
+Let's look at a few useful queries we can do just with the tailscale tables.
+
+### Unseen devices
+
+Find all devices that weren't seen more than 30 days:
+
+```sql
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:60]:
```suggestion
```sql copy
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+    tailnet    |             name              |        id         |      last_seen      
+---------------+-------------------------------+-------------------+---------------------
+ cloudquery.io | example-name.tail341.ts.net   | 12345678082367896 | 2023-01-07 12:43:18
+```
+
+### Key expiry disabled
+
+Find all devices with key expiary disabled
+
+```sql
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:74]:
```suggestion
```sql copy
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+    tailnet    |             name              |        id         |      last_seen      
+---------------+-------------------------------+-------------------+---------------------
+ cloudquery.io | example-name.tail341.ts.net   | 12345678082367896 | 2023-01-07 12:43:18
+```
+
+### Distribution of devices
+
+Learn about how tailscale client are distributed in your org across devices
+
+```sql
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:88]:
```suggestion
```sql copy
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+
+### Key expiry disabled
+
+Find all devices with key expiary disabled
+
+```sql
+select name, id, last_seen from tailscale_devices where key_expiry_disabled;
+```
+
+```
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:78]:
```suggestion
```text
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+### Distribution of devices
+
+Learn about how tailscale client are distributed in your org across devices
+
+```sql
+select count(*), os from tailscale_devices group by os;
+```
+
+```
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:92]:
```suggestion
```text
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+   70  | window
+```
+
+### Find all devices of disabled Okta users
+
+Other cool things once you have Tailscale configuration in your database/datalake is that you can join and query it with other CloudQuery [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview).
+
+For example the following will show all devices of users that were deactivated on Okta.
+
+```
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:106]:
```suggestion
```sql copy
```

**Review by erezrokah — COMMENTED** [review]
Added a few comments. Looks great 🚀

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Introduction
+
+Tailscale source plugin for CloudQuery is now officially maintained by Tailscale - Big thanks for the Tailscale team!.
+
+With the new Tailscale plugin you can now easily query Tailscale config and bring config data to your infrastructure/operational lake
+so you can have all your infrastructure data in one place and correlate between different services.
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:19]:
```suggestion
With the new Tailscale plugin you can now easily query Tailscale config and bring config data to your infrastructure/operational lake. 
This way you can have all your infrastructure data in one place and correlate between different services.
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+
+As part of its initial release, the Tailscale plugin supports pulling data for the following APIs:
+ - [`tailscale_acls`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_acls.md) table
+ - [`tailscale_devices`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_devices.md) table
+ - [`tailscale_device_routes`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_device_routes.md) table
+ - [`tailscale_dns_nameservers`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_dns_nameservers.md) table
+ - [`tailscale_dns_searchpaths`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_dns_searchpaths.md) table
+ - [`tailscale_keys`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/tailscale_keys.md) table
+
+Let's look at a few useful queries
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:30]:
```suggestion
Let's look at a few useful queries.
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+
+## Configuration
+
+First let's quickly look at the source configuration so the plugin can authenticate with the new OAuth Client
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:34]:
```suggestion
First, let's quickly look at the source configuration so the plugin. The example below syncs the `tailscale_devices` table to PostgreSQL, making use of the new OAuth client to authenticate.
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+
+```
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:36]:
```suggestion
```yaml copy
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+    tailnet: "cloudquery.io"
+```
+
+## Example Queries
+
+Let's look at a few useful queries we can do just with the Tailscale tables.
+
+### Unseen devices
+
+Find all devices that weren't seen more than 30 days:
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:58]:
```suggestion
Find all devices that haven't been seen in more than 30 days:
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 19]
```diff
@@ -0,0 +1,116 @@
...
+
+```sql
+select name, id, last_seen from tailscale_devices where last_seen < NOW() - INTERVAL '30 DAY';
+```
+
+```
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:64]:
```suggestion
```text
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 20]
```diff
@@ -0,0 +1,116 @@
...
+
+### Key expiry disabled
+
+Find all devices with key expiry disabled
+
+```sql
+select name, id, last_seen from tailscale_devices where key_expiry_disabled;
+```
+
+```
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:78]:
```suggestion
```text
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 21]
```diff
@@ -0,0 +1,116 @@
...
+
+### Distribution of devices
+
+Learn about how Tailscale client are distributed in your org across devices
+
+```sql
+select count(*), os from tailscale_devices group by os;
+```
+
+```
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:92]:
```suggestion
```text
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 22]
```diff
@@ -0,0 +1,116 @@
...
+ count |  os   
+-------+-------
+   102 | macOS
+   103 | linux
+   70  | window
+```
+
+### Find all devices of disabled Okta users
+
+Other cool things once you have Tailscale configuration in your database/datalake is that you can join and query it with other CloudQuery [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview).
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:102]:
```suggestion
Other cool things once you have Tailscale configuration in your database/datalake is that you can join and query it with data from other CloudQuery [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview).
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 23]
```diff
@@ -0,0 +1,116 @@
...
+
+For example the following will show all devices of users that were deactivated on Okta.
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:104]:
```suggestion
For example, the following will show all devices of users that were deactivated on Okta:
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 24]
```diff
@@ -0,0 +1,116 @@
...
+
+```
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:106]:
```suggestion
```sql copy
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 25]
```diff
@@ -0,0 +1,116 @@
...
+select td.tailnet, td.name, td.id, td.user, td.last_seen
+from tailscale_devices td
+    left join okta_users ou 
+        on td.user = ou.profile->>'email'
+where ou.stats != 'ACTIVE'
+```
+
+## Summary
+
+You can slice and dice Tailscale config data in many ways but this should give you a great starting point!
```
Comment (hermanschaaf) [inline website/pages/blog/cloudquery_tailscale.mdx:116]:
```suggestion
Using data from the new Tailscale source plugin you can slice and dice your Tailscale data in many ways, but the examples in this post should give you a good starting point.
```

**Review by erezrokah — APPROVED** [review]

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 26]
```diff
@@ -0,0 +1,116 @@
...
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Introduction
+
+The Tailscale source plugin for CloudQuery is now officially maintained by Tailscale—big thanks to the Tailscale team!
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:16]:
```suggestion
The [Tailscale](https://tailscale.com/) source plugin for CloudQuery is now officially maintained by Tailscale—big thanks to the Tailscale team!
```

File: website/pages/blog/cloudquery_tailscale.mdx [hunk 27]
```diff
@@ -0,0 +1,116 @@
...
+ count |  os   
+-------+-------
+   102 | macOS
+   103 | linux
+   70  | window
+```
+
+### Find all devices of disabled Okta users
+
+Other cool things once you have Tailscale configuration in your database/datalake is that you can join and query it with data from other CloudQuery [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview).
```
Comment (erezrokah) [inline website/pages/blog/cloudquery_tailscale.mdx:102]:
```suggestion
Other cool things once you have Tailscale configuration in your database/datalake is that you can join and query it with data from other CloudQuery [source plugins](/docs/plugins/sources/overview).
```

**Comment by candiduslynx** [comment]
@yevgenypats any updates?

**Comment by yevgenypats** [comment]
Waiting on Tailscale team - it's in progress I think.

**Pull request merged**